services/order.py

```diff
@@ -0,0 +1,28 @@
+from django.db import transaction
+
+from db.models import Ticket, Order, User
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> None:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
it is better to use get_user_model() method instead of User
fix this everywhere